src/pyfme/aircrafts/cessna_172.py

```diff
@@ -330,10 +337,13 @@ def _calculate_thrust_forces_moments(self):
 
         return Ft
 
-    def calculate_forces_and_moments(self):
-        Ft = self._calculate_thrust_forces_moments()
-        L, D, Y, l, m, n = self._calculate_aero_forces_moments()
-        Fg = self.gravity_force
+    def calculate_forces_and_moments(self, system, environment, controls):

```

Comment:
The only place `self` is used in this method is for retrieving `alpha` and `beta`. But perhaps those belong to the `System` too? And if so, what's the point of having this method in the `Aircraft`?

Reply:
Sorry, I amend myself: it is also used to access the mass and the controls. Still, seems a bit unnatural.